### PR TITLE
Prepare to release azure-spring-bom and azure-spring-cloud-dependencies.

### PR DESCRIPTION
**Pull request by chenrujun**
Prepare to release azure-spring-bom and azure-spring-cloud-dependencies.

### CODE REVIEW

**Comment by chenrujun** [comment]
Hi, @stliu , @saragluna , please help to review this PR when you have time.

**Review by saragluna — APPROVED** [review]
LGTM.

File: sdk/boms/azure-spring-boot-bom/pom.xml [hunk 1]
```diff
@@ -57,37 +50,44 @@
 
   <dependencyManagement>
     <dependencies>
+      <dependency>
+        <groupId>com.azure</groupId>
+        <artifactId>azure-sdk-bom</artifactId>
+        <version>1.0.4</version>
```
Comment (saragluna) [inline sdk/boms/azure-spring-boot-bom/pom.xml:56]: why don't we have a property entry for this version?

Comment (chenrujun) [reply sdk/boms/azure-spring-boot-bom/pom.xml:56]: Use number here is more straightforward. Just like azure-sdk-bom did.

**Pull request merged**